test.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import statsmodels.stats.api as sms
+import pickle
+
+from modelling.feat_engineer.data_processing import make_seq
+from tensorflow.keras.models import load_model
+
+
+def get_conf_int_for(sample_id, X_test, y_test, features=None, num_samples=250, show=True, bins=25):
+    X_sample = np.tile(X_test[sample_id], (num_samples, 1, 1))
+    if features is not None:
+        features = np.tile(features[sample_id], (num_samples, 1))
+        ps = model.predict([X_sample, features])
+    else:
+        ps = model.predict(X_sample)
+    if show:
+        plt.hist(ps, bins=bins)
+        plt.show()
+    begin, end = sms.DescrStatsW(ps.flatten()).tconfint_mean()
+    print('Confidence interval: ', begin, end)
+    print('Mean: ', np.mean([begin, end]))
+    print('True: ', y_test[sample_id])
+
```

Comment:
It would be great if this could return a [named tuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) with the lower bound, mean, and upper bound. 

```python
Prediction = namedtuple('Prediction', ['lower', 'mean', 'upper'])
return Prediction(lower=<value>, mean=<value>, upper=<value>)
```

Then later we can write a scoring function that takes in `y_test` and evaluates the fraction of predictions that contain the ground truth in their confidence interval.